### PR TITLE
fix(renderer): fix tdf renderer bug

### PR DESCRIPTION
**Pull request by etkmao**
# Pre-PR Checklist

- [x] I added/updated relevant documentation.
- [x] I followed the [Convention Commit](https://conventionalcommits.org/) guideline with maximum 72 characters to submit commit message.
- [x] I squashed the repeated code commits.
- [x] I signed the [CLA].
- [x] I added/updated test cases to check the change I am making.
- [x] All existing and new tests are passing.

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/Hippy?pullRequest=2552) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/Hippy?pullRequest=2552) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: etkmao<br/>:x: lukehuang<br/><hr/>**lukehuang** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/Hippy?pullRequest=2552) it.</sub>

**Review by canluhuang — APPROVED** [review]
LGTM

File: renderer/tdf/common/src/renderer/tdf/viewnode/scroll_view_node.cc [hunk 1]
```diff
@@ -127,18 +185,22 @@ void ScrollViewNode::InitDragEndListener() {
...
-      Init##Event##Listener();                                                                       \
-    }                                                                                                \
-    listener##id_ = scroll_view->Add##Event##Listener(listener);                                     \
-  }
-
-// Called by CreateView
-RegisterListener(ScrollStart, scroll_start_listener_) RegisterListener(ScrollUpdate, scroll_update_listener_)
-    RegisterListener(ScrollEnd, scroll_end_listener_) RegisterListener(DragStart, drag_start_listener_)
-        RegisterListener(DragEnd, drag_end_listener_)
+#define RegisterListener(Event, listener)                                                             \
```
Comment (medns) [inline renderer/tdf/common/src/renderer/tdf/viewnode/scroll_view_node.cc:188]: Capitalize macro names

File: renderer/tdf/common/src/renderer/tdf/viewnode/view_node.cc [hunk 2]
```diff
@@ -18,13 +18,17 @@
  * limitations under the License.
  */
 
+#pragma clang diagnostic ignored "-Wdeprecated-copy"
```
Comment (medns) [inline renderer/tdf/common/src/renderer/tdf/viewnode/view_node.cc:21]: move to actual use place and add push & pop syntax

File: framework/android/src/main/cpp/minimum_exports.lst [hunk 3]
```diff
@@ -25,6 +25,8 @@
   global:
     JNI_OnLoad;
     JNI_OnUnload;
+    Java_com_tencent_tdf_keyboard_KeyboardChannel_*;
```
Comment (medns) [inline framework/android/src/main/cpp/minimum_exports.lst:28]: Using JNI dynamic registration instead of static exposure

**Review by medns — CHANGES_REQUESTED** [review]

File: renderer/tdf/common/src/renderer/tdf/view/view_pager.cc [hunk 4]
```diff
@@ -23,6 +23,7 @@
 #include "core/common/point.h"
 #include "core/common/rect.h"
 #include "core/common/scalar.h"
+#include "footstone/logging.h"
```
[the remark this answers is not in the document]
Comment (etkmao) [reply renderer/tdf/common/src/renderer/tdf/view/view_pager.cc]: fixed

File: renderer/tdf/common/include/renderer/tdf/tdf_render_manager.h [hunk 5]
```diff
@@ -19,6 +19,8 @@
  */
 
 #pragma once
+#pragma clang diagnostic ignored "-Wdeprecated-copy"
+#pragma clang diagnostic ignored "-Wextra-semi"
```
[the remark this answers is not in the document]
Comment (canluhuang) [reply renderer/tdf/common/include/renderer/tdf/tdf_render_manager.h]: fixed

File: renderer/tdf/common/include/renderer/tdf/viewnode/base64_image_loader.h [hunk 6]
```diff
@@ -19,11 +19,12 @@
  */
 
 #pragma once
+#pragma clang diagnostic ignored "-Wextra-semi"
```
[the remark this answers is not in the document]
Comment (canluhuang) [reply renderer/tdf/common/include/renderer/tdf/viewnode/base64_image_loader.h]: fixed

File: renderer/tdf/common/include/renderer/tdf/viewnode/image_view_node.h [hunk 7]
```diff
@@ -19,6 +19,7 @@
  */
 
 #pragma once
+#pragma clang diagnostic ignored "-Wdeprecated-copy"
```
[the remark this answers is not in the document]
Comment (canluhuang) [reply renderer/tdf/common/include/renderer/tdf/viewnode/image_view_node.h]: fixed

File: renderer/tdf/common/include/renderer/tdf/viewnode/net_image_loader.h [hunk 8]
```diff
@@ -19,11 +19,12 @@
  */
 
 #pragma once
+#pragma clang diagnostic ignored "-Wextra-semi"
```
[the remark this answers is not in the document]
Comment (canluhuang) [reply renderer/tdf/common/include/renderer/tdf/viewnode/net_image_loader.h:23]: fixed

File: renderer/tdf/common/include/renderer/tdf/viewnode/text_input_node.h [hunk 9]
```diff
@@ -72,7 +76,7 @@ constexpr const char kMaxLength[] = "maxLength";                          // int
 constexpr const char kMultiline[] = "multiline";                          // boolean
 constexpr const char kNumberOfLines[] = "numberOfLines";                  // int
 constexpr const char kOnBlur[] = "onBlur";                                // boolean
-constexpr const char kOnChangeText[] = "onChangeText";                    // boolean
+constexpr const char kOnChangeText[] = "changetext";                      // boolean
```
[the remark this answers is not in the document]
Comment (canluhuang) [reply renderer/tdf/common/include/renderer/tdf/viewnode/text_input_node.h:78]: Yes, the correct event name is "changetext"

File: renderer/tdf/common/src/renderer/tdf/viewnode/scroll_view_node.cc (hunk 1 above)
Comment (canluhuang) [reply renderer/tdf/common/src/renderer/tdf/viewnode/scroll_view_node.cc]: fixed

File: renderer/tdf/common/src/renderer/tdf/viewnode/view_node.cc (hunk 2 above)
Comment (canluhuang) [reply renderer/tdf/common/src/renderer/tdf/viewnode/view_node.cc]: fixed

File: framework/android/src/main/cpp/minimum_exports.lst (hunk 3 above)
Comment (canluhuang) [reply framework/android/src/main/cpp/minimum_exports.lst]: fixed

File: renderer/tdf/common/include/renderer/tdf/viewnode/text_input_node.h (hunk 9 above)
[the remark this answers is not in the document]
Comment (medns) [reply renderer/tdf/common/include/renderer/tdf/viewnode/text_input_node.h:78]: Does that mean that all events with `on` prefix in the context need to be modified?

File: renderer/tdf/common/CMakeLists.txt [hunk 10]
```diff
@@ -47,6 +47,11 @@ GlobalPackages_Add(footstone)
 target_link_libraries(${PROJECT_NAME} PUBLIC footstone)
 # endregion
 
+# region vfs
```
Comment (medns) [inline renderer/tdf/common/CMakeLists.txt:50]: There is no `vfs` related part in the current commit, is there a necessary dep here?

**Review by medns — CHANGES_REQUESTED** [review]

File: renderer/tdf/common/include/renderer/tdf/viewnode/text_input_node.h (hunk 9 above)
[the remark this answers is not in the document]
Comment (canluhuang) [reply renderer/tdf/common/include/renderer/tdf/viewnode/text_input_node.h:78]: Other events have the same problem: 
OnBlur -> blur
OnContentSizeChange -> contentSizeChange

File: renderer/tdf/common/CMakeLists.txt (hunk 10 above)
Comment (canluhuang) [reply renderer/tdf/common/CMakeLists.txt:50]: This later commit([91498c4](https://github.com/Tencent/Hippy/pull/2552/commits/91498c4d2b9c2e03130300d1117489282237add9)) has used the vfs' UriLoader

**Review by medns — APPROVED** [review]
LGTM

**Review by churchill-zhang — APPROVED** [review]

**Review by siguangli — APPROVED** [review]

**Review by zoomchan-cxj — APPROVED** [review]
LGTM

**Pull request merged**